ui/src/dashboards/components/DashboardsCardGrid.scss

```diff
@@ -3,12 +3,25 @@
   ------------------------------------------------------------------------------
 */
 
+.dashboards-index__page-contents {
+  .cf-dapper-scrollbars--content {
+    height: 100% !important;
+  }
+  .cf-page-contents__fixed {
+    height: 100%;
+  }
+  .cf-resource-list {
+    height: 100%;
+  }
+}
+
 .dashboards-card-grid {
    display: grid;
    grid-template-columns: 1fr;
    grid-template-rows: 150px 150px 150px;
    grid-column-gap: $ix-marg-a;
    grid-row-gap: $ix-marg-a;
+   height: 100%;
```

Comment:
You’re 100%

Reply:
it was the worst drilling 100% height all the way to the frame component, or else it just chooses whatever height it wants to be